test/services/bulkstatus.service.test.js

```diff
@@ -81,6 +81,13 @@ describe('Dynamic X-Progress logic', () => {
         expect(response.headers['x-progress']).toEqual('60.00% Done');
       });
   });
-
+  test('check operationOutcome includes the number of resources when available', async () => {
+    await supertest(server.app)
+      .get('/4_0_1/bulkstatus/COMPLETED_REQUEST_WITH_RESOURCE_COUNT')
+      .expect(200)
+      .then(response => {
+        expect(response.body.outcome[0].type).toEqual('OperationOutcome');
```

Comment:
This doesn't seem to be checking that there's some count of resources available? It's just checking the type is OperationOutcome